doc/source/whatsnew/v0.24.0.txt

```diff
@@ -179,7 +179,7 @@ Other Enhancements
 - :class:`IntervalIndex` has gained the :meth:`~IntervalIndex.set_closed` method to change the existing ``closed`` value (:issue:`21670`)
 - :func:`~DataFrame.to_csv`, :func:`~Series.to_csv`, :func:`~DataFrame.to_json`, and :func:`~Series.to_json` now support ``compression='infer'`` to infer compression based on filename extension (:issue:`15008`).
   The default compression for ``to_csv``, ``to_json``, and ``to_pickle`` methods has been updated to ``'infer'`` (:issue:`22004`).
-- :func:`to_sql` now supports writing ``TIMESTAMP WITH TIME ZONE`` columns (:issue:`9086`)
+- :meth:`DataFrame.to_sql` now supports writing ``TIMESTAMP WITH TIME ZONE`` columns (:issue:`9086`)
```

Comment:
I think it would be good to mention what this means in practice.

So if one had now a tz-aware column, it was converted to naive UTC, and you got back naive UTC. So the difference is now that you will get back tz-aware UTC ? (and of course that the schema of the sql table now includes 'with time zone')

But so for databases that don't support timezones. Previously we converted to UTC so users got also back naive UTC, while now they will get back naive local? 

(trying to fully understand the consequences)